.cirrus.yml

```diff
@@ -0,0 +1,17 @@
+freebsd_instance:
+  image: freebsd-12-0-release-amd64
+task:
+  name: FreeBSD
+  env:
+    matrix:
+      - JULIA_VERSION: 1.0
+      - JULIA_VERSION: 1.1
+      - JULIA_VERSION: 1.2
+      - JULIA_VERSION: 1.3
+      - JULIA_VERSION: nightly
```

Comment:
Is there a `allow_failures` for cirrus?

Reply:
There is, but the syntax isn't very clear. Nightly works for other packages though, which is confusing.

Reply:
The error is not very descriptive.